modules/helm-disabling-helm-chart-repository-in-the-cluster.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/helm/configuring-custom-helm-chart-repositories.adoc
+
+[id="disabling-helm-chart-repository-in-the-cluster_{context}"]
+= Disabling Helm Chart repository in the cluster
+
+You can disable Helm Charts in the catalog by adding `disabled` property in the `HelmChartRepository` custom resource. 
+
+.Procedure
+
+* To disable a recently added Helm Chart repository by using CLI, add the `disabled: true` flag to the custom resource. For example, to remove an Azure sample chart repository, run:
++
+----
+$ cat <<EOF | oc apply -f -
+apiVersion: helm.openshift.io/v1beta1
+kind: HelmChartRepository
+metadata:
+  name: azure-sample-repo
+spec:
+  connectionConfig:
+   url:https://raw.githubusercontent.com/Azure-Samples/helm-charts/master/docs
+  disabled: true
+EOF
+----
+
+*  To disable a recently added Helm Chart repository by using Web Console:
++
+. Go to *Custom Resource Definitions* and search for `HelmChartRepository` custom resource.
+
+. Go to *Instances*, find the repository you want to disable and click on its name.
+
+. Go to the *YAML* tab, add the `disabled: true` flag in the `spec` section and click `Save`.
+
+. The repository is now disabled and will not appear in the catalog.  
```

Comment:
There is no user action, so this line can't be presented as a step. I suggest nesting it under the previous step.
```suggestion
+
The repository is now disabled and will not appear in the catalog.  
```